tools/licenses/lib/main.dart

```diff
@@ -2854,16 +2854,19 @@ Future<void> _collectLicensesForComponent(_RepositoryDirectory componentRoot, {
   }
 
   sink.writeln('UNUSED LICENSES:\n');
-  final List<String> unusedLicenses = licenses
+  final List<License> rawUnusedLicenses = licenses
```

Comment:
pedantic nit: The term `raw` here is a bit ambiguous/unclear to me. I'm assuming this just means it's a list of licence objects rather than strings. 

I wonder if we should keep the existing name for symmetry with the `List<License> usedLicenses` below and instead update the `List<String> unusedLicenses` below to `unusedLicenseOutput`, then perhaps `output` -> `usedLicenseOutput` below?

Reply:
my next patch deletes this entire block of code so since you said this is optional i'm going to skip it :-)